src/annotator/plugin/bucket-bar.js

```diff
@@ -205,17 +173,9 @@ export default class BucketBar extends Delegator {
 
   _buildTabs() {
```

Comment:
`_buildTabs` is a misleading name IMO as this function merely updates the properties of existing tab elements, which must have already been constructed by `_update`. Since `_buildTab` is only called in one place at the end of `_update`, I think the code would be just as readable if the body of this method were moved into the `_update` function above - and it would likely make other changes easier.

Reply:
This one is unchanged from the legacy implementation of this module. I'll catch it on my next pass which will update DOM details (I want to attack the a11y issues here).